nilearn/glm/first_level/first_level.py

```diff
@@ -421,6 +421,13 @@ def fit(self, run_imgs, events=None, confounds=None,
             respective run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
+        sample_masks : Any type compatible with numpy-array indexing, or
+            list of indices, optional
+            shape of array: (number of scans - number of volumes removed, )
+            Indices of retained volumes. Masks the niimgs along time/fourth
+            dimension to perform scrubbing (remove volumes with high motion)
+            and/or non-steady-state volumes.
```

Comment:
```suggestion
            dimension to perform scrubbing (remove volumes with high motion
            and/or non-steady-state volumes).
```

Reply:
Scrubbing by definition, is removing volumes with high motion. Hence I don't think we should move the bracket.

Reply:
That's fair. In that case it still needs to be updated to be grammatically correct.

```suggestion
            dimension to perform scrubbing (remove volumes with high motion)
            and/or remove non-steady-state volumes.
```

Reply:
Just wondering: wouldn't it be more handy to give  a binary vector with length `n_scans` for  `sample_mask` ?

Reply:
Yeah let's open an issue to keep track of this...